packages/mongodb-chatbot-ui/src/messageLinks.ts

```diff
@@ -35,3 +40,27 @@ export function getMessageLinks(
     ? formatReferences(messageData.references, { tck: options.tck })
     : undefined;
 }
+
+export function makePrioritizeCurrentMongoDbReferenceDomain(): SortReferences {
+  const currentDomain = getCurrentPageUrl();
+  if (!currentDomain) {
+    // If we can't determine the current domain (e.g. on the server) then the sort is a no-op
+    return () => 0;
+  }
+  const prioritizableDomains = [
+    new URL("https://mongodb.com/docs"),
```

Comment:
aside: i always find the `URL` API quite confusing to work with and need to look up the API ref every time i see it